_pages/en/it-picture-long-term.md

```diff
@@ -6,12 +6,55 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## Citizen
 
+### Government IoT Integration 
+
+ESDC does not develop products or services that compete with the private sector. IoT devices will begin to permeate all aspects of life and ESDC should integrate itself into the ecosystem. 
+
+An example of the repercussions of such an implementation can be found [here](https://github.com/sara-sabr/ITStrategy/blob/master/wip-tec/en/human-development-life-cycle.md).
+
+In a service based environment, success is not measured by the number of users your service attracts, but rather the number of users amassed by the services that leverage your service. 
+
+For more details on it's implementation, please refer to the [Business section](##Businesses)
+
+### Modern Service Delivery 
+
+When was the last time you called the customer service of your favourite digital service? 
+
+Within 10 years ESDC should cease to inflict it's layering, processes, and complexities, onto it's users -- the citizens. As a government entity we are responsible to ensure the ease of use from the entirety of Canada's citizenry. 
+
+For those comfortable with consuming services digitally, which is becoming increasingly large majority of the population, physical interaction with government offices should be an option -- not a necessity. For regions which are still needing of attention to onboard digitally, different methods of service interaction need be available. Cost savings from automation and process improvement should be passed on to provide increased attention to areas where service uptake is lagging (*To-do of areas lacking use of services needed*)
+
+*Mode 1* 
+- Group: The growing number of citizens comfortable with an entirely digital 
+- Action: Develop digital services
+*Mode 2*
+- Group: Those unable or unwilling to use digital services in densely populated areas
+- Action: Implement initiatives to onboard users onto digital services 
+*Mode 3* 
+- Group: Those in areas lacking sufficient, or convenient access, to digital services 
+- Action: Maintain legacy centres, with a focus on digital education alongside offering services
```

Comment:
If "Lack of Internet Access" is included in the scope, we could add in Action something along the lines of supporting GC/CRTC initiatives toward increasing availability of Internet services in these areas.